aptos-move/aptos-vm/src/data_cache.rs

```diff
@@ -45,33 +47,23 @@ use std::{
 /// track of incremental changes is vital to the consistency of the data store and the system.
 pub struct StateViewCache<'a, S> {
     data_view: &'a S,
-    data_map: BTreeMap<StateKey, Option<Vec<u8>>>,
+    data_map: DataMapKind<'a>,
```

Comment:
there's existing type for this -> std::borrow::Cow

Reply:
Sweet!